src/app/authenticate.service.ts

```diff
@@ -34,6 +35,10 @@ export class AuthenticateService {
       );
   }
 
+  getUsers():Observable<UserDTO[]>{
+    return this.httpClient.get<UserDTO[]>((`http://localhost:8080/api/users`))
```

Comment:
I'm storing the spring boot url in the environment file

Reply:
done